temporian/core/operators/window/base.py

```diff
@@ -98,8 +92,13 @@ def build_op_definition(cls) -> pb.OperatorDef:
     def operator_def_key(cls) -> str:
         """Get the key of the operator definition."""
 
-    @classmethod
+    @abstractmethod
+    def output_features(
+        self, event: Event, sampling: Sampling
+    ) -> list[Feature]:
+        """Get the list of features in the output event."""
```

Comment:
wouldn't `output_features_dtypes(cls, input_event: Event) -> List[DType]` or similar be enough?